src/main/java/http/response/Response.java

```diff
@@ -8,34 +8,42 @@
     private OutputStream output;
     private int statusCode;
     private String statusMessage;
-    private Map<String,String> headers = new HashMap();
+    private Map<String, String> headers = new HashMap();
     private byte[] body;
 
-    public Response (OutputStream output){
+    public Response(OutputStream output) {
         this.output = output;
     }
 
 
-   public void setStatus(int code, String message){
+    public void setStatus(int code, String message) {
         this.statusCode = code;
         this.statusMessage = message;
-   }
+    }
 
-   public void setHeaders (String key, String value){
-        this.headers.put(key,value);
+    public String getStatusMessage() {
+        return this.statusMessage;
+    }
+    public int getStatusCode() {
+        return this.statusCode;
+    }
+    public void setHeaders(String key, String value) {
```

Comment:
very small nitpick, but this should be called `addHeaders` - the reason is that `set` suggests that the whole headers hashmap will be replaced (which is not a case here), while `add` that a header will be added (like you do it)

Reply:
👍